### ISSUE TITLE
Problems with <leader> key and remapping in the latest version

### ISSUE THREAD

**Issue by ihodex** [VSCodeVim/Vim#6821]
**Describe the bug**

With the latest release, I have a problem with the next remapping

```
	"vim.normalModeKeyBindingsNonRecursive": [
		{
			"before": [
				"<C-a>"
			],
			"after": [
				"leader",
				"leader",
				"s"
			]
		}
	],
```

Instead of easy motion triggering, I probably face a regular `s`

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 1.21.4
- VSCode version: 1.57.1
- OS: Windows 10

**Comment by numToStr** [comment]
I am facing the same issue with `easymotion` in the `v1.21.4`. But everything works fine in `v1.21.3`.

**Comment by CCIBB** [comment]
This should be a bug in lastest version **[1.21.4]** when use **esaymotion** default motion key-mapping like `<leader><leader>s` which also dose not work. I'm rollback to version 1.21.3 and everything works fine.

**Comment by IsaacShelton** [comment]
Same issue for me

I use `,` for `<leader>` which doesn't work as of `1.21.4`, but rolling but to `1.21.3` everything seems to work fine.

**Comment by J-Fields** [comment]
Sorry for the regression! This is fixed on 1.21.5, which is now available - please upgrade.
